test/unit/renderer/viz/expressions/buckets.test.js

```diff
@@ -14,6 +16,227 @@ describe('src/renderer/viz/expressions/buckets', () => {
     });
 
     describe('eval', () => {
-        // TODO
+        let bucketExpression;
+
+        describe('when input type is category', () => {
+            const METADATA = new Metadata({
+                properties: {
+                    city: {
+                        type: 'category',
+                        categories: [
+                            { city: 'Murcia' }, 
```

Comment:
the API for this I think is `categories: [{name: 'Murcia'}...` without repeating the property name `city`

Reply:
Nope, after testing it, it needs to be the name of the property, right?

Reply:
I don't think so :sweat_smile: https://github.com/CartoDB/carto-vl/blob/master/src/renderer/Metadata.js#L22 and https://github.com/CartoDB/carto-vl/blob/master/test/unit/renderer/viz/expressions/belongs.test.js#L10 Could you recheck the implementation too?

Reply:
Ok, I will change that :) I may need to change some of the specs.